test/mlir/accelerators/nnpa/module_op_be/lit.local.cfg

```diff
@@ -0,0 +1,7 @@
+root = config.root
+# enable when onnx-mlir driver works well with nnpa.
+config.unsupported = True
```

Comment:
Should this be true sometimes?

Reply:
Yes, once `onnx-mlir` command works with NNPAAccelerator. Currently NNPAAccelerator cannot be self-register.